gateway-sp-comms/src/single_sp.rs

```diff
@@ -762,6 +764,86 @@ impl SingleSp {
             data
         })
     }
+
+    /// Instruct the SP that a reset_component_trigger will be coming with a
+    /// boot image selection policy setting.
+    ///
+    /// This is part of a two-phase reset process. MGS should set a
+    /// `reset_component_prepare()` followed by `reset_component_trigger()`. Internally,
+    /// `reset_component_trigger()` continues to send the reset trigger message until the
+    /// SP responds with an error that it wasn't expecting it, at which point we
+    /// assume a reset has happened. In critical situations (e.g., updates),
+    /// callers should verify through a separate channel that the operation they
+    /// needed the reset for has happened (e.g., checking the SP's version, in
+    /// the case of updates).
+    pub async fn reset_component_prepare(
+        &self,
+        component: SpComponent,
+    ) -> Result<()> {
+        debug!(
+            self.log,
+            "sending ResetComponentPrepare component:{component:?}"
+        );
```

Comment:
I don't think we need the two `debug!` logs in this method - all messages sent and received are already logged (at the `trace!` level) by `Inner`'s handlers for rpc requests.

Reply:
removed.